commands/betflip.js

```diff
@@ -11,36 +15,42 @@ module.exports = {
     owner_only: false,
     affect_config: true,
     action: function(message, config) {
-        user = update_dabs(message.author, config)
         content = message.content.toLowerCase().split(" ")
         bonus = 1
-        if (content[1] === "all"){
-            amount = user.dabs
-            choice = content[2]
-            bonus = 2
-        } else if (content[2] === "all") {
-            amount = user.dabs
+        if (content.length !== 3){
```

Comment:
Just ignore everything past the third word, no need to limit how people interact with the bot.